app/code/Magento/Catalog/Api/Data/ProductInterface.php

```diff
@@ -105,6 +105,13 @@ public function setAttributeSetId($attributeSetId);
     public function getPrice();
 
     /**
+     * Product final price
+     *
+     * @return float|null
+     */
+    public function getFinalPrice();
```

Comment:
Would you consider it if I make a new PR for it, or should I create my own module ?